### ISSUE TITLE
In order to download products from Oracle Technology Network you must agree to the OTN license terms

### ISSUE THREAD

**Issue by olyv** [vrivellino/chef-java_se#10]
Hi,

I am using java_se cookbook to download specific jdk version (I added `depends 'java_se', '~> 8.0'` to my metadata.rb). I used this recipe about a month ago but now it fails because it can't download package from Oracle 

```
Recipe: java_se::_macosx_install
  * ruby_block[fetch http://download.oracle.com/otn-pub/java/jdk/8u162-b12/0da788060d494f5095bf8624735fa2f1/jdk-8u162-macosx-x64.dmg] action run
    
    ================================================================================
    Error executing action `run` on resource 'ruby_block[fetch http://download.oracle.com/otn-pub/java/jdk/8u162-b12/0da788060d494f5095bf8624735fa2f1/jdk-8u162-macosx-x64.dmg]'
    ================================================================================

```

I assume this is related to not accepted license agreement:

![image](https://user-images.githubusercontent.com/5982003/40664498-5a2c4018-635b-11e8-8ea8-1b8e89677ce8.png)

Could you please advise if you experienced this issue?

**Comment by vrivellino** [comment]
@olyv: It appears a new version of Java 8 has been published (8u171). When that happens, the cookbook has to be updated to use it. If you wish to continue to use 8u162, then you'd need to set the `uri` attribute: https://github.com/vrivellino/chef-java_se#attributes

I'll get an update out as soon as possible!

**Comment by vrivellino** [comment]
@olyv I just published cookbook version 8.172.0. You may need to run `berks update java_se` to update Berksfile.lock so that the new version gets picked up.

**Comment by olyv** [comment]
Hi,
as I can see  after `berks update java_se` it is 

```
Fetching cookbook index from https://supermarket.chef.io...
Using automation-setup (0.1.0) from source at .
Installing java_se (8.172.0)
```

And I could install jdk 8 successfully. In my case I am happy with version 8 either 162 or 172.

Thanks a lot for quick answer!
